File: SofiForce.Models/Models/SearchModels/SalesOrderControlSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class SalesOrderControlSearchModel : BaseSearchModel
    {
		public Int32? ClientId { get; set; }
		public string SalesCode { get; set; }
		public string ClientCode { get; set; }
		public Int32? BranchId { get; set; }
		public Int32? RepresentativeId { get; set; }
		public Int32? StoreId { get; set; }
		public Int32? PriorityTypeId { get; set; }
		public DateTime? SalesDateFrom { get; set; }
		public DateTime? SalesDateTo { get; set; }
		public Int32? SalesOrderTypeId { get; set; }

		public Int32? SalesOrderStatusId { get; set; }
		public Int32? RejectedOnly { get; set; }
	}
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/SalesOrderControlSearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface SalesOrderControlSearchModel extends BaseSearchModel {
    clientId: number | null;
    salesCode: string;
    clientCode: string;
    branchId: number | null;
    branchCode:string;
    representativeId: number | null;
    representativeCode: string | null;

    storeId: number | null;
    storeCode: string;

    priorityTypeId: number | null;
    salesDateFrom: Date | null;
    salesDateTo: Date | null;
    salesOrderTypeId: number | null;
    salesOrderStatusId: number | null;
    rejectedOnly: number | null;
}